tests/integration/test_error.sh

```diff
@@ -16,6 +16,15 @@ set -e
 
 set -x
 
-test ${BAZEL_EXIT_CODE} != 0
+test ${BAZEL_EXIT_CODE} = 0
```

Comment:
I think you also need to change line 12 to `bazel test` instead of `bazel build`

And do that in all other test scripts